contracts/social-tips/examples/schema.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_schema::{export_schema, remove_schemas};
+use schemars::schema_for;
+use social_tips::msg::{
+    ExecuteMsg, InstantiateMsg, QueryConfigResponse, QueryMsg, QueryPendingTipsResponse,
+    QueryUnclaimedSentTipsResponse,
+};
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InstantiateMsg), &out_dir);
+    export_schema(&schema_for!(ExecuteMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+    export_schema(&schema_for!(QueryPendingTipsResponse), &out_dir);
+    export_schema(&schema_for!(QueryUnclaimedSentTipsResponse), &out_dir);
+    export_schema(&schema_for!(QueryConfigResponse), &out_dir);
```

Comment:
The reason why you change `write_api` to `export_schema` is because the `ts-codegen` does not work properly, right?
Maybe we should try [new ts codegen](https://github.com/CosmWasm/ts-codegen) built by Cosmwasm team.

Reply:
Should we revert the changes of other contract schemas to `export_schema` as well before updating ts-codegen?

Reply:
Yes, I have reverted the changes for that reason. Yes the changes should be reverted to use the new `write_api`.